armi/operators/settingsValidation.py

```diff
@@ -219,6 +226,38 @@ def run(self, cs=None):
                 )
             runLog.debug("{} has finished querying.".format(self.__class__.__name__))
 
+        # decide what to do with original settings file
+        if correctionsMade:
+            oldFilePath = "{}_old.yaml".format(self.cs.path.split(".yaml")[0])
+            if self._csRelativePathExists(oldFilePath):
+                overwrite_file = prompt(
+                    'INSPECTOR: At least one corrective query has been resolved and an "_old.yaml" settings file {} already exists.'.format(
+                        oldFilePath
+                    ),
+                    'Overwrite "_old.yaml" settings file to preserve originally submitted settings file?',
+                    "YES_NO",
+                    "NO_DEFAULT",
+                    "CANCEL",
+                )
+                if overwrite_file:
+                    os.replace(tempFilePath, oldFilePath)
+            else:
+                preserve_settings = prompt(
+                    "INSPECTOR: At least one corrective query has been resolved.",
+                    "Preserve originally submitted settings in: {}?".format(
+                        oldFilePath
+                    ),
+                    "YES_NO",
+                    "NO_DEFAULT",
+                    "CANCEL",
+                )
+                if preserve_settings:
+                    os.replace(tempFilePath, oldFilePath)
```

Comment:
here is where i would recommend putting hte shUtil instead of replace and then you dont need the code above.